lib/graphql/convert_dev_to_test_store.graphql

```diff
@@ -0,0 +1,10 @@
+mutation convertDevToTestStore($input: ConvertDevToTestStoreInput!) {
```

Comment:
Does this input type `ConvertDevToTestStoreInput` need to be defined somewhere? I couldn't find it.